stustapay/core/service/order.py

```diff
@@ -65,90 +93,115 @@ async def create_order(
             count,
             order_id,
         )
+        order = await self._fetch_order(conn=conn, order_id=order_id)
+        if order is None:
+            raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR)
```

Comment:
there should be a message? and is this also logged in the server console? because internal errors should be well visible